code/modules/reagents/Chemistry-Machinery.dm

```diff
@@ -187,7 +187,7 @@
 	if(machine_stat & BROKEN)
 		return
 	var/mob/living/carbon/human/H = user
```

Comment:
```suggestion
```

Reply:
That won't really change must and it doesn't affect this PR